docs/docs/rest-api/public/api/v2/groups.raml

```diff
@@ -0,0 +1,376 @@
+    Application groups can contain other application groups.
+  is: [ secured ]
+  body:
+    application/json:
+      example: !include examples/group.json
+      schema: Group
+  responses:
+    200:
+      description:
+        A deployment is started which has a unique deployment identifier.
+        The related deployment can be fetched from the /v2/deployments endpoint.
+        If the deployement is gone from the list of deployments, than this means it is finished.
+        As long as the deployment runs, the effect of that change operation is visible only partially.
+      body:
+        application/json:
+          example: !include examples/deployments_result.json
+    400:
+      description: The group definition provided in the body is not valid.
+      body:
+        application/json:
+          example: |
+            {"message":"Invalid JSON","details":[{"path":"/id","errors":["error.expected.jsstring"]}]}
+    409:
+      description: There is an already deployed group with this name
+      body:
+        application/json:
+          example: |
+            {"message":"Group / is already created. Use PUT to change this group."}
+    422:
+      description: The entity send can not be preocessed, since there are validation errors
+      body:
+        application/json:
+          example: |
+            {
+              "message":"Bean is not valid",
+              "errors": [
+                {
+                  "attribute":"apps[0].id",
+                  "error":"identifier /app is not child of /group. Hint: use relative paths."
+                }
+              ]
+            }
+
+
+###
+# Delete the root group
+#
+delete:
+  description:
+    Destroy a group. All data about that group and all associated applications will be deleted.
+    The failure or success of the action is signalled via event. There is a group_change_success and group_change_failed with
+    the given version.
+  is: [ secured ]
+  responses:
+    200:
+      description:
+        A deployment is started which has a unique deployment identifier.
+        The related deployment can be fetched from the /v2/deployments endpoint.
+        If the deployement is gone from the list of deployments, than this means it is finished.
+        As long as the deployment runs, the effect of that change operation is visible only partially.
+      body:
+        application/json:
+          example: !include examples/deployments_result.json
+    409:
+      description:
+        There is an already running deployment, that affects one application in that group.
+        To override this deployment, use the force=true flag
+      body:
+        application/json:
+          example: |
+            {
+             "message":"App is locked by one or more deployments. Override with the option '?force=true'. View details at '/v2/deployments/<DEPLOYMENT_ID>'.",
+             "deployments":[{"id":"97c136bf-5a28-4821-9d94-480d9fbb01c8"}]
+            }
+
+
+/{group_id}:
```

Comment:
Good idea. Removed everything under {group_id} and imported the same file twice.